shared/constants/types/fs.tsx

```diff
@@ -82,22 +82,44 @@ export type TlfSyncPartial = I.RecordOf<_TlfSyncPartial>
 
 export type TlfSyncConfig = TlfSyncEnabled | TlfSyncDisabled | TlfSyncPartial
 
-export enum ConflictState {
+export enum ConflictStateType {
   None = 'none',
-  InConflictStuck = 'in-conflict-stuck',
-  InConflictNotStuck = 'in-conflict-not-stuck',
-  InManualResolution = 'in-manual-resolution',
-  Finishing = 'finishing',
+  Automatic = 'automatic',
+  ManualServerView = 'manual-server-view',
+  ManualLocalView = 'manual-local-view',
 }
 
-export type _TlfConflict = {
-  state: ConflictState
-  branch: string
+export type _ConflictStateNone = {
+  type: ConflictStateType.None
 }
-export type TlfConflict = I.RecordOf<_TlfConflict>
+export type ConflictStateNone = I.RecordOf<_ConflictStateNone>
+
+export type _ConflictStateAutomaticResolving = {
+  isStuck: boolean
+  type: ConflictStateType.Automatic
+}
+export type ConflictStateAutomaticResolving = I.RecordOf<_ConflictStateAutomaticResolving>
+
+export type _ConflictStateManualResolvingServerView = {
+  localViewTlfPaths: I.List<Path>
+  type: ConflictStateType.ManualServerView
+}
+export type ConflictStateManualResolvingServerView = I.RecordOf<_ConflictStateManualResolvingServerView>
+
+export type _ConflictStateManualResolvingLocalView = {
```

Comment:
If I split A into A and B, A gets ManualResolvingServer and B gets ManualResolvingLocal. Then what happens when there's a new conflict in A?

I feel like maybe we ought to combine ManualResolvingServer and AutomaticResolving, such that you can be in both states at once, if that makes any sense.

WDYT?

Reply:
> If I split A into A and B, A gets ManualResolvingServer and B gets ManualResolvingLocal. Then what happens when there's a new conflict in A?

I _think_ in that case A would become "automatic resolve" with isStuck being true, then you can start resolve again from A to split into C. @strib could you confirm?

Reply:
> I feel like maybe we ought to combine ManualResolvingServer and AutomaticResolving, such that you can be in both states at once, if that makes any sense.

Hmm perhaps ManualResolvingServer and the "none" case should merge instead. Without any more conflict being resolved, ManualResolvingServer is essentially "none" right?

Reply:
yeah.. how about merging all three of ManualServer, None, and Automatic, such that at any time you can have (1) a list of conflict copies and (2) a current conflict?

Reply:
Good point! I've just made this change. PTAL thanks!